src/libs/canOpenURLInSameTab/index.desktop.ts

```diff
@@ -0,0 +1 @@
+export default false;
```

Comment:
I need to add this so it won't open the expensify classic on the same tab on desktop.

Reply:
That's simpler, but I thought we prefer to use platform-specific files.

Reply:
Updated